uvahelpr_web/marketplace/urls.py

```diff
@@ -6,4 +6,5 @@
     url(r'^$', views.index, name='index'),
     url(r'^jobs/(?P<id>[0-9]+)/$', views.job_entry, name="job_entry"),
     url(r'^jobs/$', views.allJobs, name='allJobs'),
+    url(r'^home/$', views.home, name='home'),
```

Comment:
Agreed, just pick one of the views. I'd prefer the url for it match "" than "home/" because the homepage should be something very easy for the user to remember (facebook.com vs facebook.com/home).